app/controllers/projects/samples/attachments_controller.rb

```diff
@@ -4,7 +4,6 @@ module Projects
   module Samples
     # Controller actions for Project Samples Attachments
     class AttachmentsController < ApplicationController
```

Comment:
**nitpick (blocking):** Since you removed the before action here, I believe you need to update this to inherit from `Projects::ApplicationController` instead of just `ApplicationController`

Reply:
This was fixed in 7f856988fa31564ae415cb0b75ed55eac7adeddc